### PR TITLE
new icon: akka (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #920: Icon request: akka
### About the icon

Akka is a toolkit for building highly concurrent, distributed, and resilient message-driven applications for Java and Scala.

### Links and sources

+ <a href="https://akka.io/">Akka Official Page</a>
+ <a href="https://akka.io/resources/images/akka_full_color.svg">Direct Link to the SVG</a>

### Preview of the original Icon
akka_full_color

### Additional information

I can work on this Issue, but I'm a bit busy right now. If anybody else wants to work on it feel free to do so!

### PR DESCRIPTION
**Pull request by ozanuslan**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #920**

## Link to prove your SVG is correct and up-to-date.

+ <a href="https://akka.io/">Akka Official Page</a>
+ <a href="https://akka.io/resources/images/akka_full_color.svg">Direct Link to the SVG</a>
+ Original SVG preview: 
akka_full_color

@Panquesito7 could you take a look? 😊

### CODE REVIEW

**Review by fmanimashaun — APPROVED** [review]
Hey @ozanuslan 

nice work 👋

**Comment by ozanuslan** [comment]
The system doesn't seem to have registered your approval :smile: Could you re-review it? @fmanimashaun

**Review by fmanimashaun — DISMISSED** [review]
@ozanuslan 

It should be approved once some with access right review and approve!

File: devicon.json
Comment (Panquesito7) [inline devicon.json:84]:
```suggestion
            "scala",
            "open-source"
```

**Comment by Panquesito7** [comment]
Hmm, not sure it looks good when uploaded to IcoMoon. I recommend making a plain version.
If you need any help, check our [**Wiki resources**](https://github.com/devicons/devicon/wiki/SVG-Versions) or ask on our [**Discord server**](https://discord.gg/hScy8KWACQ). Thanks. 🙂

File: devicon.json
Comment (Snailedlt) [inline devicon.json:95]: The `original` and `original-wordmark` icons aren't mono-colored, so they can't be used for our fonts.
In order to fix this you need to add separate mono-colored `plain` and `plain-wordmark` icons.

Please check out our wiki posts on [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) and [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Looks good, but icons used for the font should be mono-colored (see comment below)

File: devicon.json
Comment (ozanuslan) [reply devicon.json]: Added `plain` and `plain-wordmark`. Changed `devicon.json` accordingly.

**Comment by ozanuslan** [comment]
What is your opinion on the color of plain versions? There are two 'main' colors of the logo; light blue `#15a9ce`, and dark blue `#0b5567`. I think the light blue versions would look better, but the dark blue was more dominant on the `original-wordmark` svg.

**Comment by Snailedlt** [comment]
> What is your opinion on the color of plain versions? There are two 'main' colors of the logo; light blue `#15a9ce`, and dark blue `#0b5567`. I think the light blue versions would look better, but the dark blue was more dominant on the `original-wordmark` so I picked that one.

@ozanuslan 
I'm leaning more towards the lighter color, both because it looks better and because it's used more on the main website and in the docs, but it's up to you

**Comment by ozanuslan** [comment]
I agree with you. Changing the color in `devicon.json` to `#15a9ce`.

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 
Thanks for contributing! 🙇

**Review by Panquesito7 — APPROVED** [review]
Great work! Thanks. 🚀

**Pull request merged**
